src/net.cpp

```diff
@@ -2493,7 +2561,17 @@ void CConnman::ThreadOpenConnections(const std::vector<std::string> connect)
             }
         }
 
-        CSemaphoreGrant grant(*semOutbound);
+        MaybePickPrivateBroadcastNetwork(open_private_broadcast_to,
+                                         m_private_broadcast_connections_to_open.load(),
+                                         num_private_broadcast_opened);
+
+        // Don't wait for outbound connection slot to be available if we are going
```

Comment:
Private connections are opened regardless of max connections limits, I think we could cover it in the functional test.

Reply:
The current functional test already opens the usual 10 connections and then fires a private broadcast one. Is this what you mean?

Reply:
Yes, I missed that. Nevermind.